DeepSpeech.ipynb

```diff
@@ -149,6 +150,7 @@
     "validation_step = int(os.environ.get('ds_validation_step', 1))   # TODO: Determine a reasonable value for this\n",
     "checkpoint_step = int(os.environ.get('ds_checkpoint_step', 5))   # TODO: Determine a reasonable value for this\n",
     "checkpoint_dir = os.environ.get('ds_checkpoint_dir', xdg.save_data_path('deepspeech'))\n",
+    "restore_checkpoint = bool(int(os.environ.get('ds_restore_checkpoint', 0)))\n",
```

Comment:
So no variable or 0 disables this, anything being a valid int > 0 enables it and a string throws an exception, good for me for now :)